spec/lib/miq_automation_engine/miq_ae_service_spec.rb

```diff
@@ -110,4 +110,48 @@ module MiqAeServiceSpec
       end
     end
   end
+
+  describe MiqAeService do
+    context "service models" do
+      it "expose all expected active_record models as service_models" do
+        excluded_model_names = %w(
+          AuthToken
+          AuthUseridPassword
+          Category
+          Datacenter
+          ManageIQ::Providers::BaseManager
+          ManageIQ::Providers::Kubernetes::ContainerManager::Scanning::Job
+          VmServer
+          VmSynchronize
```

Comment:
@gmcculloug it feels like this should be a constant in MiqAeService.  What do you think?